tests/main/uc20-snap-recovery/task.yaml

```diff
@@ -0,0 +1,48 @@
+summary: Integration tests for the snap-recovery binary
+
+# one system is enough, its a very specialized test for now
+systems: [ubuntu-18.04-64]
+
+restore: |
+    if [ -f loop.txt ]; then
+        losetup -d "$(cat loop.txt)"
+    fi
+
+prepare: |
+    echo "Create a fake block device image"
+    dd if=/dev/zero of=fake.img bs=1M seek=2000 count=1
```

Comment:
You can also just call `truncate --size=2GB fake.img` which seems "nicer" since it is zero-filled and sparse.

Reply:
Pretty sure the outcome is (almost) the same, note the "seek" in the dd.

Reply:
Ah, I didn't notice the seek is large and count is small.

Reply:
I updated to use truncate now to make things easier to read.